sb-common/src/main/kotlin/ru/driics/sablebot/common/service/GatewayService.kt

```
@@ -0,0 +1,4 @@
+package ru.driics.sablebot.common.service
+
+interface GatewayService {
+}
```

detekt (1.23.8)

[warning] The class or object GatewayService is empty.

(detekt.empty-blocks.EmptyClassBlock)